### ISSUE TITLE
user request: use http url as input

### ISSUE THREAD

**Issue by janwilmans** [djeedjay/DebugViewPP#215]
This is very useful for, for example,  java server-based log files.
Assuming the context is just plaintext (as it is with most logfiles) this should be pretty low-hanging fruit.

I would want to stream the file, not download it. 

After some research streaming is only possible if the server is configured to support the HTTP 1.1 RANGE command. see 14.16 Content-Range at https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html

related links:
- URLDownloadToFile  ( https://msdn.microsoft.com/en-us/library/ms775123(v=vs.85).aspx )
- http://www.codeproject.com/Articles/20181/The-Ultimate-TCP-IP-Home-Page

**Comment by janwilmans** [comment]
The output of sometimes like:
	while(1); do \
		sleep 2; \
		wget -ca -O log.txt -o /dev/null http://yoursite.com/log; \
	done

could of course be piped into debugview++

**Comment by janwilmans** [comment]
I think this should be part of the refactor done for #191

**Comment by janwilmans** [comment]
To enable drag-dropping url's to debugview, it looks like we need to drag COM support in.

The window the is registered using RegisterDragDrop has to:
- implement IDropTarget ?
- be a COM object ?

Maybe we could have a separate class like:

```
class DropTargetSupport : public CComObjectRootEx <CComSingleThreadModel>, public IDropTarget
{
    Register(HWND hwnd)
    {
        RegisterDragDrop(hwnd, this);
    }

    Unregister(HWND hwnd)
    {
        RevokeDragDrop(hwnd);
    }
}
```

see http://intercontineo.com/article/643153589/

**Comment by janwilmans** [comment]
implemented in 5156bfbc88aaea4051cf6b12335c244611112a23
